.gitignore

```
@@ -0,0 +1,3 @@
+**/__pycache__/
+**/.idea/
+**/.vscode/
```

since those two are only found in the top-level of the sources, their entries can be simplified as
```suggestion
/.idea/
/.vscode/
```